# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-mips64-xtalk-bridge.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

mips64-t2-linux-gnuabin32-ld: arch/mips/pci/pci-xtalk-bridge.o: in function `bridge_get_partnum':
pci-xtalk-bridge.c:(.text+0x384): undefined reference to `crc16'

--- linux-6.9/arch/mips/Kconfig.vanilla	2024-06-20 18:56:55.436910044 +0200
+++ linux-6.9/arch/mips/Kconfig	2024-06-20 18:55:14.057915930 +0200
@@ -1217,6 +1217,7 @@
 	bool
 
 config PCI_XTALK_BRIDGE
+	select CRC16
 	bool
 
 config NO_EXCEPT_FILL
